src/client/startup.js

```diff
@@ -147,6 +147,8 @@ sand.globalFunctions = {
 					}
 				}
 			});
+			// running this after the ajax request in case the user has analytics blocked
```

Comment:
I've run into an issue recently where ga() blocks functionality when people have google analytics urls blocked with Ghostery or uBlock. ga() would be defined, but the function would error out. So I try and avoid putting them before anything important.